From fb56a1b03af2799d38757f0f15981b77e26ef96f Mon Sep 17 00:00:00 2001
From: Rock Shi <shipengyang@nd.com.cn>
Date: Wed, 7 Dec 2022 15:10:59 +0800
Subject: [PATCH] AP9-10516 AP9 - Google Slides via Chromium unable to create
 or drag and drop

Fix Version: 1.2.1
1. there is no touchscreen feature on AP9 and we should keep this for some app(VLC)
2. we force return 5 for maxTouchPoint
---
 ui/android/java/src/org/chromium/ui/base/TouchDevice.java | 8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

diff --git a/ui/android/java/src/org/chromium/ui/base/TouchDevice.java b/ui/android/java/src/org/chromium/ui/base/TouchDevice.java
index dc7bc2abe0be0..d2dec0e39b89d 100644
--- a/ui/android/java/src/org/chromium/ui/base/TouchDevice.java
+++ b/ui/android/java/src/org/chromium/ui/base/TouchDevice.java
@@ -31,7 +31,9 @@ public class TouchDevice {
         // guarantees a minimum number of touch points. Be conservative and return the minimum,
         // checking membership from the highest class down.
 
-        if (ContextUtils.getApplicationContext().getPackageManager().hasSystemFeature(
+        // nd patch start
+        // remove for AP9-10516
+        /*if (ContextUtils.getApplicationContext().getPackageManager().hasSystemFeature(
                     PackageManager.FEATURE_TOUCHSCREEN_MULTITOUCH_JAZZHAND)) {
             return 5;
         } else if (ContextUtils.getApplicationContext().getPackageManager().hasSystemFeature(
@@ -45,7 +47,9 @@ public class TouchDevice {
             return 1;
         } else {
             return 0;
-        }
+        }*/
+        return 5;
+        // nd patch end
     }
 
     /**
-- 
2.34.1

